src/zope/error/error.py

```diff
@@ -79,20 +79,15 @@ def getPrintable(value, as_html=False):
                     xml_escape(type(value).__name__))
         if isinstance(value, bytes):
             value = _u(value, errors="zope.error.printedreplace")
-    if as_html:
-        return value
-    else:
-        return xml_escape(value)
+    return value if as_html else xml_escape(value)
+
 
 def getFormattedException(info, as_html=False):
     lines = []
     for line in format_exception(as_html=as_html, *info):
         line = getPrintable(line, as_html=as_html)
         if not line.endswith("\n"):
-            if not as_html:
-                line += "\n"
-            else:
-                line += "<br />\n"
+            line += "\n" if not as_html else "<br />\n"
```

Comment:
I would find it easier to understand without the negative, i.e. `line += "<br />\n" if as_html else "\n"`